demo/summer-2016.html

```diff
@@ -259,32 +264,6 @@
         enableLiveAutocompletion: true
     });
 
-    //we will move this code into tex_completer in next pull request and well make this beautiful
-    var HashHandler = require("ace/keyboard/hash_handler").HashHandler; 
```

Comment:
I thought it was refactored a few days ago, no? Did you forget to `git merge internship2016` into this branch?